src/roiextractors/multiimagingextractor.py

```diff
@@ -59,6 +57,17 @@ def _check_consistency_between_imaging_extractors(self):
                 len(unique_values) == 1
             ), f"{property_message} is not consistent over the files (found {unique_values})."
 
+    def _get_times(self):
+        frame_indices = np.array([*range(self._start_frames[0], self._end_frames[-1])])
+        times = self.frame_to_time(frames=frame_indices)
+
+        for extractor_index, extractor in enumerate(self._imaging_extractors):
+            if getattr(extractor, "_times") is not None:
+                to_replace = [*range(self._start_frames[extractor_index], self._end_frames[extractor_index])]
+                times[to_replace] = extractor._times
+
+        return times
```

Comment:
@CodyCBakerPhD, this is the current attempt to get times based on your recent comment. Let me know what you think about it; I'm not sure this is the best way to do it.

Reply:
Yeah, I'm fine with this! Good job!

The only alternative I can think of is to have an empty list that gets extended with the result of calls to `extractor.frame_to_time(frame_idxs=range(extractor.get_num_frames())` as you iterator over all the `extractors` in `self._imaging_extractors`, and then you cast that as an array at the end - but I don't believe this is much more efficient (if any) than what you have